spec/system/search/display_jobs_banner_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe "Display Jobs Banner spec", type: :system, js: true do
+  before do
+    stub_request(:post, "http://www.google-analytics.com/collect")
+    SiteConfig.jobs_url = "www.very_cool_jobs_website.com"
```

Comment:
we should mock this, otherwise it will have this value for any other test at the end

Reply:
+1 on mocking, good catch @rhymes.

Reply:
Good call!!!